cloud_functions/purge_on_bucket_change/src/main.py

```diff
@@ -50,15 +52,19 @@ def invalidate_for_gs_change(bucket: str, key: str, invalidator: Invalidator) ->
     #find what to purge
     paper_id = _paperid(key)
     if not paper_id:
-        logging.info(f"No purge: gs://{bucket}/{key} not related to an arxiv paper id")
+        logging.debug(f"No purge: gs://{bucket}/{key} not related to an arxiv paper id")
         return
     
     if "/pdf/" in key:
```

Comment:
You could limit it to just PDFs with
`if "/pdf/" in key and key.endswith(".pdf"):`